static/js/admin/vue-permissions-editor/src/components/staffRoles.vue

```diff
@@ -0,0 +1,270 @@
+<template>
+    <div id="staff-roles">
+        <h1 v-if="canSetPermissions">Set Staff Permissions</h1>
+        <h1 v-else>Inherited Staff Permissions</h1>
+
+        <table class="border inherited-permissions" v-if="current_staff_roles.inherited !== undefined && current_staff_roles.inherited.length > 0">
+            <thead>
+            <tr>
+                <th>Staff</th>
+                <th>Permission</th>
+                <th>Inherited from</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr v-for="inherited_staff_permission in current_staff_roles.inherited">
+                <td>{{ inherited_staff_permission.principal }}</td>
+                <td>{{ inherited_staff_permission.role }}</td>
+                <td>{{ containerName }}</td>
```

Comment:
Hmm, while this is technically correct (permissions are always inherited from the parent), I think the intent was to state specifically which parent the roles originated from. You don't have access to that info right now though, so I've created a follow up ticket to address this https://jira.lib.unc.edu/browse/BXC-2453

Reply:
Yeah, I'd meant to ask about this one.